or_slam:
  pure_localization: true
  use_sim: true
  odom_frame_id: "odom"
  base_frame_id: "base_link"
  global_frame_id: "map"
  laser_topic_name: "scan"
  trunk_topic_name: "trunk_obs"
  transform_tolerance: 0.1
  trunk_std_radius: 0.16

  initial_pose_x: 2.5
  initial_pose_y: 8
  initial_pose_a: 0
  initial_cov_xx: 0.1
  initial_cov_yy: 0.1

  #地图输出位置
  out_map_file_name: /home/cxn/myfile/orchardrover_ws/src/OrchardRover/or_making_map/slam_map.txt
  #地图读取位置
  read_map_file_name: /home/cxn/myfile/orchardrover_ws/src/OrchardRover/or_making_map/out_map.txt

  sim_sensor_odom:
    alpha1: 0.03  #yaw
    alpha2: 0.08  #x
    alpha3: 0.08  #y
    alpha4: 0.01